packages/widget_toolkit_pin/example/lib/main.dart

```diff
@@ -111,35 +110,102 @@ class MyHomePage extends StatelessWidget {
           dialogHasBottomPadding: true,
         ),
       );
+
+  String _exampleMapMessageToString(BiometricsMessage message) {
+    switch (message) {
+      case BiometricsMessage.notSetup:
+        return 'To use biometrics, you need to turn it on in your device settings!';
+
+      case BiometricsMessage.notSupported:
+        return 'You don\'t have biometric feature on your device!';
+
+      case BiometricsMessage.enabled:
+        return 'Your biometrics are enabled!';
+
+      case BiometricsMessage.disabled:
+        return 'Your biometrics are disabled!';
+    }
+  }
 }
 
 class AppPinCodeService implements PinCodeService {
-  AppPinCodeService();
+  AppPinCodeService({required this.inMemoryInstance});
+
+  final InMemoryInstance inMemoryInstance;
```

Comment:
I believe we can simplify even further the example project by removing the `InMemoryInstance`